web_monitoring/differs.py

```diff
@@ -51,11 +52,20 @@ def pagefreezer(a_url, b_url):
     # Just send PF the urls, not the whole body.
     # It is still useful that we downloaded the body because we are able to
     # validate it against the expected hash.
-    return web_monitoring.pagefreezer.compare(a_url, b_url)
+    obj = web_monitoring.pagefreezer.PageFreezer(a_url, b_url)
```

Comment:
I think our priority was setting up our own diff service which is already up (thanks to you).
What I had in mind, as a long term plan, is addition of more custom diffs written by our developers and no dependency on PF, Versionista etc which I think is pretty much along the lines of the ideas you have shared.
Also, to clarify, what I meant by our go-to diff service, was the service(s) we are currently using or relying on (PF and Versionista). So, what I meant to ask was, given that we're almost certain about not using them in the future, does it make sense to spend time on figuring out how to best use their API?